platform_versioned_docs/version-23.4/cli/commands.mdx

```diff
@@ -0,0 +1,1027 @@
+
+  ```console
+  tw workspaces add --name=shared-workspace --full-name=shared-workspace-for-all  --org=my-tower-org --visibility=SHARED
+
+    A 'SHARED' workspace 'shared-workspace' added for 'my-tower-org' organization
+  ```
+
+  :::note
+  By default, a workspace is set to private when created.
+  :::
+
+  ### List workspaces
+
+  List all the workspaces in which you are a participant:
+
+  ```console
+  tw workspaces list                      
+
+    Workspaces for default user:
+
+      Workspace ID    | Workspace Name   | Organization Name | Organization ID 
+      -----------------+------------------+-------------------+-----------------
+      26002603030407  | shared-workspace | my-tower-org      | 04303000612070  
+  ```
+
+</details>
+<details>
+  <summary>**Participants**</summary>
+
+  Run `tw participants -h` to view supported participant operations.
+
+  Manage workspace participants. 
+
+  ### List participants
+
+  ```console
+  tw participants list
+
+    Participants for 'my-tower-org/shared-workspace' workspace:
+
+      ID             | Participant Type | Name                        | Workspace Role 
+      ----------------+------------------+-----------------------------+----------------
+      45678460861822 | MEMBER           | user (user@mydomain.com) | owner          
+  ```
+
+  ### Add participants
+
+  Run `tw participants add -h` to view the required and optional fields for adding a participant.
+
+  To add a new _collaborator_ to the workspace, use the `add` subcommand. The default role assigned to a _collaborator_ is `Launch`.
+
+  See [Participant roles][participant-roles] for more information.
+
+  ```console
+  tw participants add --name=collaborator@mydomain.com --type=MEMBER                           
+
+    User 'collaborator' was added as participant to 'shared-workspace' workspace with role 'launch'
+  ```
+
+  ### Update participant roles
+
+  To update the role of a _Collaborator_ to `ADMIN` or `MAINTAIN`, use the `update` subcommand:
+
+  ```console
+  tw  participants update --name=collaborator@mydomain.com --type=COLLABORATOR --role=MAINTAIN
+
+    Participant 'collaborator@mydomain.com' has now role 'maintain' for workspace 'shared-workspace'
+  ```
+
+</details>
+<details>
+  <summary>**Teams**</summary>
+
+  Run `tw teams -h` to view supported team operations.
+
+  Manage organization teams.
+
+  ### List teams
+
+  Run `tw teams list -h` to view the required and optional fields for listing teams. 
+
+  ```console
+  tw teams list -o TestOrg2
+
+  Teams for TestOrg2 organization:
+
+     Team ID        | Team Name | Members Count Name
+    ----------------+-----------+--------------------
+     84866234211969 | Testing   | 1
+  ```
+
+  ### Add a team 
+
+  Run `tw teams add -h` to view the required and optional fields for creating a team. 
+
+  ```console 
+  tw teams add -n team1 -o TestOrg2 -d testing
+
+  A 'team1' team added for 'TestOrg2' organization
+  ```
+
+  ### Delete a team 
+
```

Comment:
Does this require a level of permissions to execute this command (like `MAINTAIN` role and above)?